test/kubernetes/e2e/features/helm/types.go

```diff
@@ -17,6 +17,12 @@ var (
 				UpgradeValues: productionRecommendationsSetup,
 			},
 		},
+		"TestChangedConfigMapTriggersRollout": {
```

Comment:
had to add these because `test/kubernetes/e2e/tests/base/base_suite.go` is now more strict with missing manifests

Reply:
reverted